### DISCUSSION TITLE
Install dependencies without optional

### DISCUSSION THREAD

**Discussion by RuBAN-GT** [yarnpkg/berry#2841] [Q&A] [answered]
Hi everyone, in my package.json I have several entries from `optionalDependencies`. 
The previous yarn v1 has an ability to ignore them using `ignore-optional` flags. 
What about the latest yarn v2? As I understood properly `install` command doesn't provide the way to control it (https://yarnpkg.com/cli/install).

To add `ignore-optional` option for install command.

Thank you!

**Comment by merceyz** [comment, marked as answer]
Duplicate of https://github.com/yarnpkg/berry/discussions/2412
See https://github.com/yarnpkg/berry/discussions/2412#discussioncomment-308895
